package.json

```diff
@@ -22,14 +22,14 @@
     "build:ui:desktop:win": "docker-compose -f docker-compose-embedding.yml run build-desktop-production && yarn build:ui:desktop:app",
     "build:ui:desktop:production": "yarn --cwd ui/desktop build:production",
     "build:ui:desktop:app": "yarn --cwd ui/desktop build:desktop",
-    "lint": "npm-run-all -p lint:*",
+    "lint": "concurrently --kill-others-on-fail \"yarn lint:addons-api\" \"yarn lint:addons-auth\" \"yarn lint:addons-core\" \"yarn lint:addons-rose\" \"yarn lint:ui-admin\" \"yarn lint:ui-desktop\"",
```

Comment:
Why are these commands listing all the packages to run instead of using the wild card syntax to run all the ones in this package.json?

Reply:
@ZedLi good catch, will fix